android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -650,6 +663,26 @@ public void pickFeature(float posX, float posY) {
             nativePickFeature(mapPointer, posX, posY, featurePickListener);
         }
     }
+    /**
+     * Set a listener for label pick events
+     * @param listener Listener to call
+     */
+    public void setLabelPickListener(LabelPickListener listener) {
+        labelPickListener = listener;
+    }
+
+    /**
+     * Query the map for labeled features at the given screen coordinates; results will be returned
+     * in a callback to the object set by {@link #setLabelPickListener(LabelPickListener)}
+     * @param posX The horizontal screen coordinate
+     * @param posY The vertical screen coordinate
+     */
+    public void pickLabels(float posX, float posY) {
```

Comment:
It seems that this is done here in the android SDK https://github.com/mapzen/android/blob/5eb635120a2ea3388c5c66f40c51ec5a276ec3ad/library/src/main/java/com/mapzen/android/graphics/MapzenMap.java#L54, isn't this what triggers the whole selection in Eraser Map?